test/framework/verify/pod.go

```diff
@@ -146,10 +146,20 @@ func (v *PodVerification) ExpectPodHaveDesiredPhase(namespace string,
 	}
 }
 
-func (v *PodVerification) WindowsPodHaveExpectedIPv4Address(pod *v1.Pod) {
+func (v *PodVerification) WindowsPodHaveIPv4Address(pod *v1.Pod) {
 	By("matching the IPv4 from annotation to the pod IP")
 	ipAddWithCidr, found := pod.Annotations["vpc.amazonaws.com/PrivateIPv4Address"]
 	Expect(found).To(BeTrue())
 	// Remove the CIDR and compare pod IP with the IP annotated from VPC Controller
 	Expect(pod.Status.PodIP).To(Equal(strings.Split(ipAddWithCidr, "/")[0]))
+
+}
+
+func (v *PodVerification) WindowsPodHaveResourceLimits(pod *v1.Pod, expected bool) {
+	_, found := pod.Spec.Containers[0].Resources.Limits[config.ResourceNameIPAddress]
```

Comment:
as discussed offline integration tests are failing here, need to check if `Spec.Containers[0].Resources.Limits != nil`